common/src/main/scala/models/Notification.scala

```diff
@@ -134,3 +136,25 @@ object ElectionNotification {
   import JsonUtils._
   implicit val jf = Json.format[ElectionNotification]
 }
+
+case class LiveEventNotification(
+  id: UUID,
+  `type`: NotificationType = LiveEventAlert,
+  sender: String,
+  title: String,
+  message: String,
+  expandedMessage: Option[String],
+  shortMessage: Option[String],
+  importance: Importance,
+  link1: Link, // where does the title for this come from?
+  link2: Link, // where does the title for this come from?
```

Comment:
I should remove the comment.  I wanted to know where the text that would displayed for each link came from.  It turns out it is baked into the app.